RELEASE-NOTES.md

```diff
@@ -1,5 +1,75 @@
 # Prefect Release Notes
 
+## Release 2.13.2
+
+### Server-side enforcement of deployment parameter schemas
+
+We've added the ability to enforce parameter schemas for deployments via the Prefect API! This feature will prevent creation of flow runs with parameters that are incompatible with deployed flow allowing you to discover errors sooner and avoid provisioning infrastructure for flow runs destined to fail.
+
+Use `enforce_parameter_schema` when deploying your flow to guard against invalid parameters:
+
+```python
+from prefect import flow
+from pydantic import BaseModel
+
+
+class Person(BaseModel):
+    name: str
+    greeting: str = "Hello"
+
+
+@flow(log_prints=True)
+def my_flow(person: Person, name: str = "world"):
+    print(f'{person.name} says, "{person.greeting}, {name}!"')
+
+
+if __name__ == "__main__":
+    my_flow.serve(
+        "testing-params",
+        enforce_parameter_schema=True,
+    )
+
+```
+
+An attempt to run the created deployment with invalid parameters will fail and give a reason the flow run cannot be created:
+```bash
+> prefect deployment run 'my-flow/testing-params' -p person='{"name": 1}'
+
+Error creating flow run: Validation failed for field 'person.name'. Failure reason: 1 is not of type 'string'
+```
```

Comment:
Would it be weird to include an example of it in `prefect deploy`? e.g.

`prefect deploy --enforce-parameter-schema`

Reply:
I added a line about how to use parameter enforcement with `prefect deploy`, but let me know if you think an example is still needed.

Reply:
Nice! Thanks